zerver/actions/create_user.py

```diff
@@ -251,7 +251,11 @@ def process_new_human_user(
     # from being sent after the user is created.
     clear_scheduled_invitation_emails(user_profile.delivery_email)
     if realm.send_welcome_emails:
-        enqueue_welcome_emails(user_profile, realm_creation)
+        enqueue_welcome_emails(user_profile)
+
+    # Schedule an initial email with the user's
+    # new account details and log-in information.
+    send_account_registered_email(user_profile, realm_creation)
```

Comment:
Oh -- I think in my ramble previously, I thought that one of the welcome emails was also immediate -- but they're the day-2 (well, "day 2") and onboarding-zulip-guide emails, so my tangent about message ordering is totally irrelevant, since they're delayed by a while.  So never mind that part!

Reply:
No I think you were correct -- the followup_days email is immediate, and this PR is recategorizing it as an account registration email. BTW @laurynmm we should finish the job of purging the misleading `followup_day1` name.

```
$ git grep followup_day1 | grep -v locale | wc
     10      31    1190
```

Reply:
Yeah, the ordering isn't super important with the current onboarding email schedule, but it's always possible that in the future a new onboarding email might be scheduled to be sent sooner than the current 2 days after registration.

For the rename of the email template, I'll do that as a follow up pull request. I think we also will want to take care of the "followup_day2" email rename in that same change. I'm thinking:
- followup_day1 -> account_registered
- followup_day2 -> onboarding_zulip_topics
- onboarding_zulip_guide, *no change*

@alexmv - Is there anything I need to be particularly aware of for that migration? For example, I'm thinking that we'd want to create new email templates (with the new names) and update the code that creates the scheduled emails to use those, then do a migration commit for existing scheduled messages that have the old template names to have the new ones, and then delete the old email templates in a final commit.

I'm not sure how that will affect the translated email strings, but it seems better to deal with extra updates in the translation process, and not have a situation where we're trying to send an email for a template that doesn't exist.

Reply:
Yeah, you're absolutely spot on that the complexity is that the old names exist in the `ScheduledEmail` table, so we need a migration to update them.

In our production deploy, we'll need to stop the worker before starting the migration on staging, but that's not something that we can address in the general migration.

Reply:
Opened #26385 as a draft/WIP of this scheduled email template rename.